beacon-chain/blockchain/receive_block.go

```diff
@@ -97,6 +98,14 @@ func (s *Service) ReceiveBlockNoPubsub(ctx context.Context, block *ethpb.BeaconB
 		return errors.Wrap(err, "could not clean up block deposits, attestations, and other operations")
 	}
 
+	// Send notification of the processed block to the state feed.
+	s.stateFeed.Send(&statefeed.Event{
+		Type: statefeed.BlockProcessed,
+		Data: &statefeed.BlockProcessedData{
+			BlockHash: bytesutil.ToBytes32(headRoot),
```

Comment:
`headRoot` is not correct here, should just be `root`

Reply:
Sorted.